pages/exercises/[lessonSlug].tsx

```diff
@@ -233,29 +227,26 @@ type ExerciseListItem = {
 }
 
 type ExerciseListProps = {
-  tabs: { text: string; url: string }[]
   onClickSolveExercises: () => void
   lessonTitle: string
   hideAnswered: boolean
   setHideAnswered: (hideAnswered: boolean) => void
   exercises: ExerciseListItem[]
+  lesson: ExerciseLesson
 }
 
 const ExerciseList = ({
-  tabs,
   onClickSolveExercises,
   lessonTitle,
   hideAnswered,
   setHideAnswered,
-  exercises
+  exercises,
+  lesson
 }: ExerciseListProps) => {
   return (
     <>
       <div className="mb-4">
-        <NavCard
-          tabSelected={tabs.findIndex(tab => tab.text === 'exercises')}
-          tabs={tabs}
-        />
+        <LessonTabs lesson={lesson} activeTab={'exercises'} />
```

Comment:
No need for brackets, change to `activeTab="exercises"`

Reply:
Actually it would be better to use an enum here instead of hardcoding "exercises", since this component is to be used as a child in multiple components

Reply:
Done.